content/en/_index.html

```diff
@@ -38,62 +38,54 @@ <h4 class="section-head">What is Kubeflow?</h4>
-		  class="card-img-top"
-		  draggable="false"
+          src="/docs/images/logos/jupyter-vscode-rlang.png"
+          class="card-img-top"
+          draggable="false"
           style="padding: 2rem;"
-          alt="TensorFlow"
+          alt="Jupyter + VSCode + RLang Logo"
         />
         <div class="card-body bg-primary-dark">
-          <h5 class="card-title text-white section-head">
-            TensorFlow model training
-          </h5>
+          <h5 class="card-title text-white section-head">Notebooks</h5>
           <p class="card-text text-white">
-            Kubeflow provides a custom <a target="_blank" rel="noopener" href="/docs/components/training/tftraining/">TensorFlow training job operator</a>
-            that you can use to train your ML model. In particular, Kubeflow's job operator can handle distributed TensorFlow training jobs. 
-            Configure the training controller to use CPUs or GPUs and to suit various cluster sizes.
+            <a target="_blank" rel="noopener" href="/docs/components/notebooks/overview/" >Kubeflow Notebooks</a> lets you run web-based development environments on your Kubernetes cluster by running them inside Pods.
           </p>
         </div>
       </div>
       <div class="card border-primary-dark">
         <img
-          src="/docs/images/logos/Models.png"
-		  class="card-img-top"
-		  draggable="false"
+          src="/docs/images/logos/katib.png"
+          class="card-img-top"
+          draggable="false"
           style="padding: 2rem;"
-          alt="Model serving image"
+          alt="Katib Logo"
         />
         <div class="card-body bg-primary-dark">
-          <h5 class="card-title text-white section-head">Model serving</h5>
+          <h5 class="card-title text-white section-head">
+            AutoML
+          </h5>
           <p class="card-text text-white">
-            Kubeflow supports a <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/tfserving_new/">TensorFlow Serving</a> container to export trained
-            TensorFlow models to Kubernetes. Kubeflow is also integrated with
-            <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/seldon/">Seldon Core</a>, an open source platform for deploying machine learning
-            models on Kubernetes, <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/tritoninferenceserver/">NVIDIA Triton Inference Server</a> for
-            maximized GPU utilization when deploying ML/DL models at scale, and <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/mlrun/">MLRun Serving</a>, an open-source serverless framework for deployment and monitoring of real-time ML/DL pipelines.
+            <a target="_blank" rel="noopener" href="/docs/components/katib/overview/">Katib</a> is a Kubernetes-native project for automated machine learning (AutoML) with support for hyperparameter tuning, early stopping and neural architecture search.
```

Comment:
Let's name it Kubeflow Katib, similar to other components.
```suggestion
            <a target="_blank" rel="noopener" href="/docs/components/katib/overview/">Kubeflow Katib</a> is a Kubernetes-native project for automated machine learning (AutoML) with support for hyperparameter tuning, early stopping and neural architecture search.
```

Reply:
@andreyvelich I was under the impression that the name of Katib was just "Katib" (given that's what its called on the [`kubeflow/katib`](https://github.com/kubeflow/katib) repo? For context, the reason we use "Kubeflow Notebooks" and "Kubeflow Pipelines" is because that's just their full names.

But if the other @kubeflow/wg-automl-leads can confirm what they want to name their component, we can change it.

Reply:
I am going to leave it as Katib, if you decide to rebrand as Kubeflow Katib, its easy to update.